src/pod.ts

```diff
@@ -56,7 +60,7 @@ export class Pod {
     defaultLocale: 'en',
     locales: ['en'],
   };
-  static DefaultConfigFile = 'amagaki.js';
+  static DefaultConfigFiles = ['amagaki.ts', 'amagaki.js'];
```

Comment:
I think the .js file should take precedence in the case that the user decides to precompile their ts file to js.

Reply:
Definitely; thanks for catching that. I had it that way but inadvertently flipped it when I was updating the docs.